### PR TITLE
test(rollup): add commit and finalize genesis batch

### PR DESCRIPTION
**Pull request by colinlyguo**
### Purpose or design rationale of this PR

This PR enhances rollup dir tests by:
1. add a commit and finalize genesis batch test.
2. enable info-level logs.
3. tweak a comment in `finetuneAccessList`.

### PR title

Your PR title must follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) (as we are doing squash merge for each PR), so it must start with one of the following [types](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type):

- [x] test: Adding missing tests or correcting existing tests

### Deployment tag versioning

Has `tag` in `common/version.go` been updated or have you added `bump-version` label to this PR?

- [x] No, this PR doesn't involve a new deployment, git tag, docker image tag

### Breaking change label

Does this PR have the `breaking-change` label?

- [x] No, this PR is not a breaking change

### CODE REVIEW

**Review by georgehao — DISMISSED** [review]

File: rollup/internal/controller/relayer/relayer_test.go [hunk 1]
```diff
@@ -25,8 +26,8 @@ var (
 
 	base *docker.App
 
-	// l2geth client
-	l2Cli *ethclient.Client
+	// geth client
+	l1Cli *ethclient.Client
```
Comment (0xmountaintop) [inline rollup/internal/controller/relayer/relayer_test.go]: why change to `l1Cli`?

Comment (colinlyguo) [reply rollup/internal/controller/relayer/relayer_test.go]: Because L2 relayer uses L1 client to connect to the L1 node.

Comment (colinlyguo) [reply rollup/internal/controller/relayer/relayer_test.go]: code snippet:
```
	// Create geth client.
	l1Cli, err = base.L1Client()
	assert.NoError(t, err)
```

```
// L1Client returns a ethclient by dialing running l1geth
func (b *App) L1Client() (*ethclient.Client, error) {
	if utils.IsNil(b.L1gethImg) {
		return nil, fmt.Errorf("l1 geth is not running")
	}
	client, err := ethclient.Dial(b.L1gethImg.Endpoint())
	if err != nil {
		return nil, err
	}
	return client, nil
}
```

Comment (colinlyguo) [reply rollup/internal/controller/relayer/relayer_test.go]: update: I see, I'll revert the change and description.

Comment (colinlyguo) [reply rollup/internal/controller/relayer/relayer_test.go]: For L1 layer, the message fetcher client connects to L1 service provider, and the sender connects to L2 service provider.

File: rollup/tests/rollup_test.go
Comment (0xmountaintop) [inline rollup/tests/rollup_test.go:1]: forget to update this file?

Comment (colinlyguo) [reply rollup/tests/rollup_test.go:1]: Yeah. and changed, too.

**Review by 0xmountaintop — APPROVED** [review]

**Review by georgehao — APPROVED** [review]

**Pull request merged**